packages/styles/scss/components/quote/_quote.scss

```
@@ -16,6 +16,14 @@
       @include carbon--make-row;
     }
 
+    &__footer {
+      .#{$prefix}--link-with-icon {
+        span {
+          max-width: calc(100% - 1.75rem);
```

Seems that the parent container is a flex box, so just a thought. The way as-is here is fine, too.

```suggestion
          flex: 1;
```

---

Thanks, Akira! I tested and worked well, will commit your suggestion.

---

My apologies, my suggestion broke things... 🙇